### PR TITLE
Python3 support

### PR DESCRIPTION
**Pull request by RaspberryPiFan**
python3 support for spirv

### CODE REVIEW

**Comment by RaspberryPiFan** [comment]
Reamke of #502

File: third_party/spiro/curves/bezfigs.py [hunk 1]
```diff
@@ -7,16 +7,16 @@
...
+    print('%%EndComments')
+    print('%%EndProlog')
+    print('%%Page: 1 1')
     if draw_box:
-        print x0, y0, 'moveto', x0, y1, 'lineto', x1, y1, 'lineto', x1, y0, 'lineto closepath stroke'
+        print(x0, y0, 'moveto', x0, y1, 'lineto', x1, y1, 'lineto', x1, y0, 'lineto closepath stroke')
 
 def eps_trailer():
-    print '%%EOF'
+    print('%%EOF'
```
Comment (penguinolog) [inline third_party/spiro/curves/bezfigs.py]: `print('%%EOF')` - not closed bracket

File: third_party/spiro/curves/fromcubic.py [hunk 2]
```diff
@@ -144,65 +144,65 @@ def test_to_pcorn():
         thresh = .1 ** i
         segs = cubic_bz_to_pcorn(bz, thresh)
         plot_segs(segs)
-        print >> sys.stderr, thresh, len(segs)
-        print '0 20 translate'
+        print(>> sys.stderr, thresh, len(segs))
```
Comment (penguinolog) [inline third_party/spiro/curves/fromcubic.py]: print redirection to stderr is made using `file=sys.stderr` keyword argument

File: third_party/spiro/curves/fromcubic.py [hunk 3]
```diff
@@ -144,65 +144,65 @@ def test_to_pcorn():
...
+            print('gsave 1 0 0 setrgbcolor')
             cmd = 'moveto'
             for i in range(100):
                 s = i * .01 * curve.arclen
                 x, y = curve.xy(s)
                 th = curve.th(s)
                 sth = 5 * sin(th)
                 cth = 5 * cos(th)
-                print x, y, cmd
+                print(x, y, cmd
```
Comment (penguinolog) [inline third_party/spiro/curves/fromcubic.py]: lost closing bracker

File: third_party/spiro/curves/mvc.py [hunk 4]
```diff
@@ -3,7 +3,7 @@
 import sys
 import random
 
-def run_mvc(k, k1, k2, k3, C, n = 100, do_print = False):
+def run_mvc(k, k1, k2, k3, C, n = 100, do_print(= False):
```
Comment (penguinolog) [inline third_party/spiro/curves/mvc.py]: syntax error

File: third_party/spiro/curves/bezfigs.py (hunk 1 above)
Comment (RaspberryPiFan) [reply third_party/spiro/curves/bezfigs.py]: fixed

**Pull request closed without merging**